torchvision/datasets/utils.py

```diff
@@ -43,14 +43,15 @@ def check_integrity(fpath: str, md5: Optional[str] = None) -> bool:
     return check_md5(fpath, md5)
 
 
-def download_url(url: str, root: str, filename: Optional[str] = None, md5: Optional[str] = None) -> None:
+def download_url(url: str, root: str, filename: Optional[str] = None, md5: Optional[str] = None, verbose: bool = True) -> None:
```

Comment:
@fepegar flake8 is complaining for the length of this line. Could you please resolve it?

@fmassa Are you OK with default being `True` or should it be `False` to ensure consistent behaviour with previous versions?

Reply:
I'll take a look at those flake8 issues. I was trying to be consistent with the rest of the file.

With the default being `True`, the behaviour doesn't change.

Reply:
> I was trying to be consistent with the rest of the file.

Oh I just saw that `download_and_extract_archive` uses an appropriate syntax for this.

Reply:
Done. Thanks for reviewing, @datumbox!